opennms-webapp/src/main/webapp/index.jsp

```diff
@@ -52,8 +52,20 @@
 
 	<!-- Middle Column -->
 	<div class="col-md-6" id="index-contentmiddle">
-		<% String centerUrl = System.getProperty("org.opennms.web.console.centerUrl", "/includes/categories-box.jsp"); %>
-		<jsp:include page="<%=centerUrl%>" flush="false" />
+		<%
+			String centerUrl = System.getProperty("org.opennms.web.console.centerUrl", "/includes/categories-box.jsp");
+
+			String[] centerUrlArr = centerUrl.split(",");
+
+			for(String centerUrlItem : centerUrlArr) {
+		%>
+
+		<jsp:include page="<%=centerUrlItem%>" flush="false" />
```

Comment:
please :) indentation as a member of the for statement, makes the code easier to read.